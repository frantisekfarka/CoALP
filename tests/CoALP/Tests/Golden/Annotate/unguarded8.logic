q(G,0):-p(G,0).
p(0,G):-q(0,G).
