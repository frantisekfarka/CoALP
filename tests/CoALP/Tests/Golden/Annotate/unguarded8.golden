q(V_2, 0, transform-func-1(V_-4)) :- p(V_2, 0, V_-4).
p(0, V_3, transform-func-2(V_-5)) :- q(0, V_3, V_-5).
